feat: only report newly introduced duplicate dependencies (#172)
e18e/action-dependency-diff: README.md

- 🔒 **Package trust levels** - Detects decreases in package trust levels (provenance and trusted publisher status)
- 📈 **Dependency growth** - Warns when dependency count increases significantly
- 📦 **Install size** - Warns when package size increases significantly

> - 🔄 **Duplicate versions** - Detects packages which gain newly introduced duplicate versions

Reviewer: ```suggestion
- 🔄 **Duplicate versions** - Detects when multiple versions of a package are introduced
```

| `pr-number`            | The number of the pull request to comment on                                                                               | Yes      | `${{ github.event.pull_request.number }}` |
| `dependency-threshold` | Threshold for warning about significant increase in number of dependencies                                                 | No       | `10`                                      |
| `size-threshold`       | Threshold (in bytes) for warning about significant increase in package size                                                | No       | `100000`                                  |

> | `duplicate-threshold`  | Threshold for warning about packages which gain newly introduced duplicate versions                                        | No       | `1`                                       |

Reviewer: ```suggestion
| `duplicate-threshold`  | Threshold (number of duplicates) for warning about newly introduced duplicate packages                                     | No       | `1`                                       |
```
